content/docs/get_started/login_requiring.md

```diff
@@ -0,0 +1,166 @@
+              <div class="card comment">
+               <p><span class="user">{{User}}</span> <span class="date">{{Datetime}}</span></p>
+               <p>{{Text}}</p>
+              </div>
+            </div>
+      </template>
+</div>
+[/sourcecode]
+
+Notice how the template is wrapped in a div with the `submit-success` attribute. We are only checking if the value of the comment is not empty in this sample; in case it’s empty we return an error that would cause to execute the code:
+
+[sourcecode:html]
+<div submit-error>
+<template type="amp-mustache">
+            Error! Looks like something went wrong with your comment, please try to submit it again.
+      </template>
+</div>
+[/sourcecode]
+
+To add a little detail to it, by using the attribute required, we are enforcing on the client side the presence of a text before hitting the Comment button:
+
+<amp-img src="/static/img/enforce-comment.png" alt="Enforce comment" height="325" width="300"></amp-img>
+
+[sourcecode:html]
+<input type="text" class="data-input" name="text" placeholder="Your comment..." required>
+[/sourcecode]
+
+When you add a comment and click the submit button, you should now see something similar to the following screenshot:, you should see something similar to the snapshot below:
+
+<amp-img src="/static/img/logout-button.png" alt="Comment added" height="352" width="300"></amp-img>
+
+## Logout
+Similar to the login button, how the presence of the logout button is conditional, depending on the state of the `amp-access` component:
+
+[sourcecode:html]
+<button amp-access="loggedIn" amp-access-hide tabindex="0" on="tap:amp-access.login-sign-out" class="button-primary comment-button">Logout</button>
+[/sourcecode]
+
+The URL to be used once you click on the logout is specified in the amp-access JSON configuration, as part of the login object:
+
+[sourcecode:json]
+{
+"login": {
+          "sign-in": "https://ampbyexample.com/samples_templates/comment_section/login?rid=READER_ID&url=CANONICAL_URL",
+          "sign-out": "https://ampbyexample.com/samples_templates/comment_section/logout"
+        }
+}
+[/sourcecode]
+
+Similar to the login, when the AMPByExample server receives a logout request, it uses the return URL query parameter automatically added by the AMP library and redirects to it, adding `#success=true`. By this time, you are back on the initial page; the AMPByExample cookie previously created for the login page (called `ABE_LOGGED_IN`) would be cleared at this point.
+
+
+## Summary
+In this tutorial I explored how to combine `amp-access` and `amp-form` to realize a comment section enabled by a login. I focused mostly on how we used those 2 components more than documenting them, while describing how we implemented a sample published on [ampbyexample.com.](https://ampbyexample.com/) In order to show the interaction between the component and a server, the sample provides a simple backend implementation which uses a cookie to save the login status.
```

Comment:
Rewrite as follows:

In this tutorial, we explored how to combine `amp-access` and `amp-form` to create a comment section that requires a login. 

To learn more, visit the following resources:

- [amp-access](https://www.ampproject.org/docs/reference/components/amp-access)
- [amp-form](https://www.ampproject.org/docs/reference/components/amp-form)
- ["Comment Section" Example](https://ampbyexample.com/samples_templates/comment_section/)